src/js/utils/newsApi.js

```diff
@@ -1,40 +1,32 @@
 import axios from 'axios';
 
-export const getSources = (category = null, callback) => {
+/**
+ * @function getSources - Get sources from Api using Axois
+ * @param {string} category - Set default to null
+ * @param {function} callback
+ */
+export const getSources = (callback) => {
   const api = 'https://newsapi.org/v1/sources';
-
-  if (category === null) {
-    axios.get(api)
-      .then(response => callback(response.data.sources))
-      .catch((error) => {
-        console.log(error);
-      });
-  } else {
-    const link = api + '?category=' + category;
-    axios.get(link)
-      .then(response => callback(response.data.sources))
-      .catch((error) => {
-        console.log(error);
-      });
-  }
+  axios.get(api)
+    .then(response => callback(response.data.sources))
```

Comment:
Don't you think it will be ideal to continue the chain by using promise to handle the asynchronous behaviour of this method?